ArmEmulator/Mem.fs

```diff
@@ -699,7 +715,7 @@ module Mem
 
     ///Updates the Memory in the DataPath (Adds something
     /// to memory). Called by DCDexec and FILLexec
-    let updateMemoryDataPath (inputRecord: labelInstr) (dP: DataPath<'INS>) =
+    let updateMemoryDataPath (inputRecord: LabelInstr) (dP: DataPath<'INS>) =
         let dataValList = 
```

Comment:
Not sure about this nested let, when they're not functions.. leave it for another PR though.